appveyor.yml

```diff
@@ -26,11 +26,11 @@ build:
   verbosity: minimal
 after_build:
 - cmd: >-
-    cpack -NSIS --verbose
+    cpack -C %CONFIGURATION% --verbose --config ./CPackConfig.cmake
```

Comment:
Is there file CPackConfig.cmake in repo?

Reply:
it gets generated as part of the cmake process when `include (CPack)` is called in the CMakeLists.txt file